shared/js/background/classes/ad-click-attribution-policy.js

```diff
@@ -143,6 +161,8 @@ export class AdClick {
      */
     shouldPropagateAdClickForNewTab (tab) {
         if (tab.site.baseDomain === this.adBaseDomain) {
+            this.adClickDNR.rule.condition.tabIds.push(tab._tabState.tabId)
+            this.adClickDNR.updateAdClickDNR()
```

Comment:
Do we need this here?

I suspect if the tab has been navigated the existing rule should work or should be removed?

Reply:
Here or do we need this at all? We need to add the new tab id to the existing rule for it to work.

Reply:
But then we have two tabId's within a single AdClick that's within the Tab object.

- If you're trying to minimise rules to DNR then you'll need to use a different global to store them under.
- Essentially your Tab object would be: `{tabId: 12, adClick: {adClickDNR: {tabIds: [12, 14]}}, ...}` which doesn't make much sense.

I think this should be handled in the new Tab being constructed and creating a new AdClick object.